manim/utils/deprecation.py

```diff
@@ -102,6 +102,8 @@ def deprecated(
     --------
     Basic usage::
 
+        from ..utils.deprecation import deprecated
```

Comment:
I'd recommend to write this as an absolute import (to avoid having to worry about the relative position of the file the deprecation is added to vs the deprecation module).

```suggestion
        from manim.utils.deprecation import deprecated
```